content/projects/angular-mocking/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Mocking with Angular
+ready: True
+prerequisites:
+  hard: ["topics/angular-testing-cucumber",
+         "topics/angular-mocking"]
+  soft: []
+---
```

Comment:
This wont pass the linter right now. 
In order to have the funcxtionality we need, it's best to be explicit and consistent. 

```
available_options: 
- typescript
- angular
submission_type: repo
```

Reply:
Done